Lib/importlib/_bootstrap_external.py

```diff
@@ -450,6 +450,7 @@ def _write_atomic(path, data, mode=0o666):
 #     Python 3.13a1 3552 (Remove LOAD_FAST__LOAD_CONST and LOAD_CONST__LOAD_FAST)
 #     Python 3.13a1 3553 (Add SET_FUNCTION_ATTRIBUTE)
 #     Python 3.13a1 3554 (more efficient bytecodes for f-strings)
+#     Python 3.13a1 3555 (generate specialized opcodes metadata from bytecodes.c)
```

Comment:
Does the instruction set change?
If not, then I would have thought that shouldn't be necessary

Reply:
`opcode_targets.h` does change, so updating the magic number is probably safer.